README.md

```diff
@@ -72,6 +72,18 @@ The `stack_rm` orb removes a stack and its configuration..
 | force | boolean | false | Whether or not to force deletion of the stack, leaving behind any resources managed by the stack. | 
 | working_directory | string | . | The relative working directory to run `pulumi` from. | 
 
+### pulumi/stack_output
+
+The `stack_output` sets a stack's output property to an environment variable.
+
+| Parameter         | type    | default     | description    |
+|-------------------|---------|-------------|----------------|
+| stack           | string  | (none)      | Name of the stack to remove. |
+| property_name           | string  | (none)      | Property name of the stack output to set to the environment variable. |
+| env_var           | string  | (none)      | Name of the environment variable to set the output property's value to. |
+| log_to_stderr | boolean | true | Log to stderr instead of to files. | 
```

Comment:
This flag isn't necessary. `--logtostderr` is just a flag to the `glog` logging library, and is useful when debugging. See https://github.com/golang/glog for more context. But we can just ignore it for the purposes of our orbs.

Reply:
Thanks for clearing it out! Understood.